src/lib/routes/client-api/metrics.test.ts

```diff
@@ -192,6 +192,7 @@ test('should set lastSeen on toggle', async () => {
         .post('/api/client/metrics')
         .send({
             appName: 'demo',
+            instanceId: '1',
```

Comment:
Should make the test work without adding a field here, just to make sure we're not breaking something.